### PR TITLE
Update filtering to work properly

### PR DESCRIPTION
**Pull request by odinuge**

### CODE REVIEW

File: lego/apps/users/fields.py [hunk 1]
```diff
@@ -27,6 +27,19 @@ def __init__(self, field_kwargs=None, **kwargs):
...
 
 
+class PublicUserWithGroupsField(serializers.PrimaryKeyRelatedField):
+    def use_pk_only_optimization(self):
+        return False
+
+    def to_representation(self, value):
+        from lego.apps.users.serializers.users import (
+            PublicUserWithAbakusGroupsSerializer,
+        )
```
Comment (LudvigHz) [inline lego/apps/users/fields.py:37]: Any reason not to import this at the top of the file?

**Review by LudvigHz — APPROVED** [review]
LGTM

File: lego/apps/users/fields.py (hunk 1 above)
Comment (odinuge) [reply lego/apps/users/fields.py:37]: Just copy pasta from the other one, but I suspect it is because of circular deps of some kind, but not 100% sure tho

Comment (LudvigHz) [reply lego/apps/users/fields.py:37]: Jup, checks out :ok_hand:

**Pull request merged**